components/html-block/html-block.js

```diff
@@ -250,8 +271,22 @@ class HtmlBlock extends RtlMixin(LitElement) {
 	}
 
 	async _handleSlotChange(e) {
-		if (!e.target) return;
-		await this._render(e.target);
+		if (!e.target || !this.shadowRoot) return;
+		const slot = this.shadowRoot.querySelector('slot');
+		const slottedNodes = slot.assignedNodes({ flatten: true });
+
+		if (this._hasSlottedElements(slottedNodes)) {
+			this._hasSlottedContent = true;
+			await this.updateComplete;
```

Comment:
Under what circumstance do we need to wait for `updateComplete`? Looks like this is a new addition... did you run into an issue without it?

Reply:
Nope! This was a holdover from when I was trying to render the `html` in the `render` method, at which point we did need to wait for that to happen before trying to run the renderers against it. I can remove this.